Assets/Src/Scripts/SeedCalc/CuttingBoard.cs

```diff
@@ -447,6 +493,18 @@ private class SlideAnimConfig {
                                                          new UnityEngine.Vector2(texOffsetX, 0));
     }
 
+    private void ScrollRainbowToLeftEnd() {
+      float texOffsetX = _rainbowTexInitOffsetX;
```

Comment:
No need texOffsetX variable? Use `_rainbowTexInitOffsetX` in following function call directly?

Reply:
Deleted the varlable.